config/testnet.json5

```
@@ -1,7 +1,10 @@
 {
   blockchain: {
-    //contractAddress : ''
+    contractAddress : '0x9A74f110586971345a396C74228094A04f5a5eA6',
     networkId: 31,
+    eventsEmitter: {
+      startingBlock: 1131351
```

```suggestion
      startingBlock: 1131351
      confirmations: 3
```
this should be sufficient for testnet